src/components/reminder-form/reminder-form.tsx

```diff
@@ -3,33 +3,56 @@ import React, { FormEvent, useState } from "react";
 
 const Form = ({ }: {}) => {
   const [taskName, setTaskName] = useState('');
-  const [recurring, setRecurring] = useState(false);
+  const [taskMessage, setTaskMessage] = useState('');
+  const [reminderType, setReminderType] = useState('');
+  const [reminderDate, setReminderDate] = useState('');
+  const [recurring, setRecurring] = useState(true);
```

Comment:
I would go with false as default

Reply:
For some reason when I did that, it did not work. Not sure why.

Reply:
Have tried turning it off and on again? :smile: